app/monitor/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3-alpine
+FROM python:3.10.0-alpine
```

Comment:
Why change from `3-alpine` to `3.10.0-alpine`?
Doesn't `3-alpine` link to the latest release?

Reply:
this is a comment from Ajilesh, we want to keep in sync with CDLZ who have updated to 3.10.0-alpine. :-)